src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/ExtractOriginalAlignmentRecordsByNameSparkIntegrationTest.java

```diff
@@ -10,44 +10,61 @@
 import org.broadinstitute.hellbender.utils.read.GATKRead;
 import org.broadinstitute.hellbender.utils.test.BaseTest;
 import org.testng.Assert;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
 import java.io.File;
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.List;
+import java.nio.file.Path;
+import java.util.*;
 import java.util.stream.Collectors;
 
 public final class ExtractOriginalAlignmentRecordsByNameSparkIntegrationTest extends CommandLineProgramTest {
 
-    @Test(groups = "sv")
-    public void testExtractOriginalAlignmentRecordsByNameSparkRunnableLocal() throws IOException {
+    @DataProvider
+    private Object[][] createTestData() throws IOException {
+        final List<Object[]> data = new ArrayList<>(20);
 
         final File tempWorkingDir = BaseTest.createTempDir("extractOriginalAlignmentRecordsByNameSparkIntegrationTest");
 
         FileUtils.writeLines(new File(tempWorkingDir, "names.txt"), Collections.singleton("asm013903:tig00002"));
 
         final SAMFileHeader expectedHeader;
-        final List<SAMRecord> expectedRecords;
+        List<SAMRecord> expectedRecords;
```

Comment:
Another nitpick: consider renaming this to final List<SAMRecord> expectedRecordsNormalMatch  (or similar)
and later creating List<SAMRecord> expectedRecordsInvertedMatch (or similar)
The reason being it took a bit of parsing for me to figure out why there were two blocks here that looked (at first glance) identical.

Reply:
done